### ISSUE TITLE
groupie-tracker subject

### ISSUE THREAD

**Issue by SSMM0498** [01-edu/public#2058]
groupie-tracker
---
Hi, I haven't understood those questions:

- Does the server present all the needed handlers and patterns for the http requests?
- Does the event system run as asynchronous? (usage of go routines and channels)
- Does the event/action responds as expected?

**Comment by mathisen99** [comment]
"Does the server present all the needed handlers and patterns for the http requests?"
This means: "Does the computer that's providing our information (the 'server') have all the right instructions and responses ready to handle the different requests it gets from other computers (the 'http requests')? For example, if another computer asks for a webpage or some data, does our server know what to do?"

"Does the event system run as asynchronous? (usage of go routines and channels)"
This is asking: "Does our system handle multiple tasks at the same time without waiting for each one to finish before starting the next? In the Go programming language, this is done using 'go routines' and 'channels'."

"Does the event/action responds as expected?"
Put simply, this asks: "When something happens in our system (an 'event' or 'action'), does our system react the way we want it to?" For instance, if a user clicks a button in our software, does the software do what it's supposed to do?

**Comment by SSMM0498** [comment]
Thank you

**Comment by MSilva95** [comment]
Thank you @mathisen99!
